CYLGame/www/static/player.js

```diff
@@ -92,6 +92,11 @@ class Player {
         .click(() => {
           this.on_faster_click();
         }));
+      this.btn_bar.append($("<button>")
+        .text(">|")
```

Comment:
Rather than approximating these symbols, do we want to just use the unicode? (perhaps ⏭️ for this one)

Reply:
IMHO the whole player could use a little UX tlc, but at least for now it matches the others

Reply:
Works for me! I just noticed that I _do_ in fact need to approve for you to merge, so consider this my blessing

Reply:
I just want to make sure you notice that the vertical bar character isn't vertically centered with respect to the greater than symbol. https://xkcd.com/1015/ ;)

Reply:
Perhaps it should be https://decodeunicode.org/en/u+23d0 instead 🤔

Reply:
![image](https://user-images.githubusercontent.com/8886072/96808876-5afb8580-13df-11eb-9ed5-eedf5dadbd5d.png)
It actually looks better 😮